src/languages/es-ES.ts

```diff
@@ -1367,6 +1367,15 @@ export default class extends Language {
 					a temporary ban.`,
 			examples: ['@Pete', '@Pete Spamming all channels.', '@Pete Spamming all channels, for 24 hours.']
 		}),
+		COMMAND_DEHOIST_DESCRIPTION: 'Convierta a todos a no izados',
+		COMMAND_DEHOIST_EXTENDED: builder.display('dehoist', {
+			extendedHelp: `El acto de izar implica agregar un personaje con un valor de 64 y menos delante de su apodo para
+				aparecer más arriba en la lista de miembros.
+				Este comando reemplaza con un carácter especial que los arrastra al final de la lista.
+				Ja, obtener un montón de esto, señor \`! !\`
```

Comment:
Sincerely, I can't think of a way to translate this line to Spanish and make it not sound offensive.